src/components/ParticipantInfo/ParticipantInfo.tsx

```diff
@@ -22,6 +22,7 @@ const borderWidth = 2;
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
     container: {
+      isolation: 'isolate',
```

Comment:
Also in this file can you fix 
```
margin: `${GRID_VIEW_MARGIN} - borderWidth`,` in the dominantSpeaker class?
```

And I also don't think that `filter: none` is needed on the video element styling?

Reply:
I wonder if that `filter:none` somehow got merged from when I was playing around with blurring the video when tracks switch off 🤔